### PR TITLE
Update navbar, bg-img, rules and regulations and footer,  add FAQs, f…

### PR DESCRIPTION
**Pull request by Saketh1499**
…lowchart img, aicte_logo

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -156,11 +156,11 @@ <h4 class="display-5 section-title">What is the Synfig Studio 2D animation Hacka
          </div>
          <div class="mx-auto bd-highlight px-3">
             <p class="text-justify">During the Hackathon the participants will create an original 2D animated short movie of maximum of 5 mins duration using Synfig Studio. You should make the animation (graphics, audio, etc.) using ONLY open source software. You can make the narration in (----) Indian languages for your animation. 
-            </p>
-            <!-- <div class="p-2 pt-1 pb-1 text-center">
-               <figure><img src="image/Process_flow5.png" class="img-fluid" style="width:650px;" alt="process_flow">
+            </p><br><br>
+            <div class="p-2 pt-1 pb-1 text-center">
+               <figure><img src="image/flowchart.png" class="img-fluid" style="width:650px;" alt="process_flow">
```
Comment (Sashi20) [inline index.html]: Move flowchart to "Eligibility to participate" section

File: index.html [hunk 2]
```diff
@@ -66,21 +66,21 @@
...
-            </li>
+            </li> -->
             <!-- <li class="nav-item">
                <a href="results.html" class="nav-link">Results</a>
             </li> -->
-             <!-- <li class="nav-item">
-          <a class="nav-link" href="schedule.html">Schedule</a>
-            </li> -->
+             <li class="nav-item">
+               <a class="nav-link" href="#">Schedule</a>
```
Comment (Sashi20) [inline index.html:82]: Remove from the navbar

File: index.html [hunk 3]
```diff
@@ -392,25 +399,30 @@ <h4 class="display-5 section-title">Contact Us <i class="far fa-envelope"></i></
...
-                  <li><a class="text-muted" href="https://rcloud.fossee.in" target="_blank">R on Cloud</a></li>
+                  <li><a class="text-muted" href="https://www.synfig.org/" target="_blank">Synfig.org</a></li>
+                  <li><a class="text-muted" href="https://spoken-tutorial.org" target="_blank">Spoken Tutorial</a></li>
+                  <li><a class="text-muted" href="https://forums.fossee.in/" target="_blank">Fossee Forum</a></li>
+                  <li><a class="text-muted" href="https://forums.spoken-tutorial.org" target="_blank">Spoken Tutorial Forum</a></li>
                </ul>
             </div>
             <div class="col-sm-3">
-               <h5>Resources</h5>
+               <h5>Socail Media</h5>
```
Comment (Sashi20) [inline index.html:419]: Remove social media links in the footer

**Review by Sashi20 — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Saketh1499) [reply index.html:82]: Done

File: index.html (hunk 3 above)
Comment (Saketh1499) [reply index.html:419]: Done

File: index.html (hunk 1 above)
Comment (Saketh1499) [reply index.html]: Done

**Pull request merged**